src/readers/morphologySWC.cpp

```diff
@@ -198,11 +198,11 @@ class SWCBuilder
         // 2 1 x (y-r) z r  1
         // 3 1 x (y+r) z r  1
 
-        float x = root.point[0];
-        float y = root.point[1];
-        float z = root.point[2];
-        float d = root.diameter;
-        float r = root.diameter / 2.f;
+        floatType x = root.point[0];
+        floatType y = root.point[1];
+        floatType z = root.point[2];
+        floatType d = root.diameter;
+        floatType r = root.diameter / static_cast<morphio::floatType>(2);
```

Comment:
By now there could be a global
```
constexpr morphio::floatType TWO = 2
```
wherever you define `floatType`, and this would read a little nicer. Or use Roman numerals.